modules/addNewBook.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+const addmybooks = document.querySelector('.addmybooks');
+const contactInfo = document.querySelector('.contact-info');
+const bookshelf = document.querySelector('.bookshelf');
+
+// showing the add my book section and hiding the rest
+// export function addNewBook() {
```

Comment:
- [ ] OPTIONAL
There is commented code block found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

Reply:
Thank you for the observation. I deleted the line.